nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/resources/default-doc-mapping.json

```diff
@@ -17,22 +17,22 @@
       }
     },
     {
-      "paths": {
+      "strings": {
         "mapping": {
           "type": "keyword",
-          "ignore_above": 32
+          "copy_to": "all_field",
+          "ignore_above": 256
         },
-        "path_match": "ecm:path@level*"
+        "match_mapping_type": "string"
       }
     },
     {
-      "strings": {
+      "paths": {
         "mapping": {
           "type": "keyword",
-          "copy_to": "all_field",
-          "ignore_above": 256
+          "ignore_above": 32
         },
-        "match_mapping_type": "string"
+        "path_match": "ecm:path@level*"
```

Comment:
Not sure why this section was moved around but it doesn't help reading the diff. Please revert this part or put it in a separate commit.

Reply:
There are 2 mappings a default and limited one in this module and complete in the nuxeo-distribution,
I have reordered this section so both versions have a similar part, 
here the diff rendering is really bad but the change is just moving "strings" before "paths" without any change

Reply:
Understood, but please keep cleanups in separate commits from fixes/features.